README.md

```diff
@@ -149,16 +149,14 @@ route:
 # We use this to mute any warning-level notifications if the same alert is
 # already critical.
 inhibit_rules:
-- source_match:
-    severity: 'critical'
-  target_match:
-    severity: 'warning'
+- source_matchers: [ severity="critical" ]
+  target_matchers: [ severity="warning" ]
   # Apply inhibition if the alertname is the same.
-  # CAUTION: 
-  #   If all label names listed in `equal` are missing 
+  # CAUTION:
+  #   If all label names listed in `equal` are missing
   #   from both the source and target alerts,
   #   the inhibition rule will apply!
-  equal: ['alertname']
+  equal: [ alertname, cluster, service ]
```

Comment:
Can we keep only alertname in this documentation? Thanks!

Reply:
As you wish